specification/webpubsub/data-plane/WebPubSub/preview/2021-05-01-preview/webpubsub.json

```diff
@@ -907,5 +916,16 @@
         }
       }
     }
+  },
+  "parameters": {
+    "Endpoint": {
+      "name": "Endpoint",
+      "description": "HTTP or HTTPS endpoint for the WebPubSub service instance.",
```

Comment:
I don't see the term 'WebPubSub' used in the documentation.
Should it be 'Web PubSub'? Or we can drop the word altogether and say 'for the service instance'.

Reply:
I just made the change